Web/Scotty/Route.hs

```diff
@@ -147,13 +153,11 @@ parseRequestBody bl s r =
                                                 (b:bs) -> return (bs, b)
             liftIO $ Parse.sinkRequestBody s rbt provider
 
-mkEnv :: forall m. MonadIO m => Request -> [Param] -> m ActionEnv
-mkEnv req captures = do
+mkEnv :: forall m. MonadIO m => Request -> [Param] -> Maybe Kilobytes ->m ActionEnv
```

Comment:
This functions should probably accept the whole `RouteOptions` record rather than just the max request body size? Because if other options are added, they'll probably get checked here, right?

Reply:
> This functions should probably accept the whole `RouteOptions` record rather than just the max request body size? Because if other options are added, they'll probably get checked here, right?

absolutely right, I changed the signature to accept `RouteOptions`